### PR TITLE
add: support for vivaldi browser

### PR DESCRIPTION
**Pull request by farhansaed**
This will add vivaldi browser

### CODE REVIEW

**Review by blimmer — APPROVED** [review]
Lgtm. I'm away from a real laptop, but will happily merge if you have a cycle to fix the conflicts. Thank you!

File: zsh-aws-vault.plugin.zsh [hunk 1]
```diff
@@ -79,8 +82,8 @@ function avli() {
   elif _using_linux ; then
     AVLI_TMP_PROFILE=$(mktemp --tmpdir -d avli.XXXXXX)
     case $browser in
-      *"chrom"*|*"brave"*)
-        (${browser} $AWS_VAULT_PL_BROWSER_LAUNCH_OPTS --no-first-run --new-window --disk-cache-dir="${AVLI_TMP_PROFILE}" --user-data-dir="${AVLI_TMP_PROFILE}" "${login_url}" 2>/dev/null && rm -rf "${AVLI_TMP_PROFILE}") &!
+      *"chrom"*|*"brave"*|*"vivaldi"*)
+        (${browser} "$AWS_VAULT_PL_BROWSER_LAUNCH_OPTS" --no-first-run --new-window --disk-cache-dir="${AVLI_TMP_PROFILE}" --user-data-dir="${AVLI_TMP_PROFILE}" "${login_url}" 2>/dev/null && rm -rf "${AVLI_TMP_PROFILE}") &!
```
Comment (blimmer) [inline zsh-aws-vault.plugin.zsh]: Per #32 I think we want to remove these quotes

Comment (farhansaed) [reply zsh-aws-vault.plugin.zsh]: 👍

**Comment by blimmer** [comment]
Thanks!

**Pull request merged**